.rubocop.yml

```diff
@@ -1,3 +1,7 @@
+AllCops:
+  Exclude:
+    - 'vendor/**/*'
+
```

Comment:
Could you split this and the `.gitignore` change to their own commit (single PR is fine)

Reply:
Done, see https://github.com/theforeman/foreman_maintain/pull/692